### DISCUSSION TITLE
gantt yaml!!

### DISCUSSION THREAD

**Discussion by adrwh** [microsoft/terminal#14576] [General]
A Gantt chart is defined in YAML on the road map page. Where can I find this software, I need this ASAP! 😀

**Comment by adrwh** [comment]
I think I find it in MermaidJS!

**Reply by j4james** [reply]
Yeah, I think so. The GitHub documentation references Mermaid here:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams

**Reply by zadjii-msft** [reply]
Yep. Mermaid is pretty legit, I've been using it for everything these days.
